Plugins/BezierSolution/BezierSpline.cs

```diff
@@ -620,9 +620,9 @@ public Vector3 GetPoint( float normalizedT )
 			{
 				// 2nd conditions isn't 'else if' because in rare occasions, floating point precision issues may arise; e.g. for normalizedT = -0.0000000149,
 				// incrementing the value by 1 results in perfect 1.0000000000 with no mantissa
```

Comment:
You can get rid of these lines. But afterwards, please make sure that the file's line endings are still **CRLF** and definitely not MIXED.